src/interfaces/chain.h

```diff
@@ -182,6 +182,9 @@ class Chain
         uint64_t max_tries,
         bool keep_script) = 0;
 
+    //! Parse confirm target.
+    virtual unsigned int parseConfirmTarget(const UniValue& value) = 0;
```

Comment:
This feels like a weird method to put in the chain interface because it's just an RPC helper function. I'd actually rather it be duplicated (though that might still require a `maxConfirmTarget` method on the interface).